src/processing/ClientBoardState.java

```diff
@@ -0,0 +1,47 @@
+package processing;
+
+import java.util.*;
+import processing.boardobject.*;
+import processing.utility.*;
+
+/**
+* This class contains all the info of the current board.
+*
+* @author Himanshu Jain
+*/
+
+public class ClientBoardState {
+	
+	//maps is an object of the BoardState containing both the maps
+	public static BoardState maps = new BoardState();
+	
+	//port Number of the Board Server
+	public static int portNumber;
+	
+	//public static Map <Identifier identifier, IChanges handler>
+	
+	public static String boardId;
+	
+	//undo and redo stacks
```

Comment:
I guess the documentation and comments are not upto the mark @himanshu2047, can you improve it.